synapse/rest/media/v1/thumbnailer.py

```diff
@@ -129,5 +129,8 @@ def crop(self, width, height, output_type):
 
     def _encode_image(self, output_image, output_type):
         output_bytes_io = BytesIO()
-        output_image.save(output_bytes_io, self.FORMATS[output_type], quality=80)
+        fmt = self.FORMATS[output_type]
+        if fmt == 'JPEG':
```

Comment:
```suggestion
        if fmt == "JPEG":
```

Reply:
It looks like this change was made, so this request can be marked as resolved.